Detectors/TPC/simulation/include/TPCSimulation/BoxClusterer.h

```diff
@@ -42,9 +41,8 @@ namespace o2{
       /// Steer conversion of points to digits
       /// @param digits Container with TPC digits
       /// @return Container with clusters
-      ClusterContainer* Process(TClonesArray *digits) override;
-      ClusterContainer* Process(std::vector<std::unique_ptr<Digit>>& digits) override;
-      
+      void Process(std::vector<o2::TPC::Digit> const & digits) override;
```

Comment:
Shouldn't this be a small `process`? I guess we never fixed it ...

Reply:
It should but I would refrain from intermixing feature changes with cosmetics. We should do a cosmetic campaign for this.